lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -554,19 +554,42 @@ struct RvalueExprVisitor {
         // Handle expressions.
         if (!listExpr->type->isSimpleBitVector()) {
           if (listExpr->type->isUnpackedArray()) {
+            if (listExpr->type->isFixedSize()) {
+              auto arrayType = dyn_cast<moore::UnpackedArrayType>(
+                  context.convertType(*listExpr->type));
+              auto elementType = arrayType.getElementType();
+              auto elementSize = elementType.getBitSize().value();
+              auto &type =
+                  listExpr->type->as<slang::ast::FixedSizeUnpackedArrayType>();
+              auto lower = type.getFixedRange().lower();
+              auto upper = type.getFixedRange().upper();
+              for (size_t i = lower; i <= upper; i++) {
+                auto value = context.convertRvalueExpression(*listExpr);
+                auto elemValue = builder.create<moore::ExtractOp>(
+                    loc, elementType, value, i);
+                if (i > lower) { // avoiding repetition of cond in the vector
+                  conditions.push_back(cond);
+                }
+                cond = builder.create<moore::EqOp>(loc, lhs, elemValue);
```

Comment:
> The inside operator uses the equality ( == ) operator on nonintegral expressions to perform the
comparison. If no match is found, the inside operator returns 1'b0. Integral expressions use the wildcard
equality (==?) operator so that an x or z bit in a value in the set is treated as a do-not-care in that bit position
(see 11.4.6).
> *— IEEE 1800-2017, §11.4.13. Set membership operator*